src/UserInput.vue

```diff
@@ -281,7 +280,7 @@ export default {
       this.file = file
     },
     _editFinish() {
-      this.store.editMessage = null
```

Comment:
Using `setState` is closer to Vuex, and allows for better debugging, but it isn't necessary.